packages/react-divider/src/components/Divider/useDividerStyles.ts

```diff
@@ -33,20 +33,16 @@ const useBaseStyles = makeStyles({
     lineHeight: tokens.lineHeightBase200,
     textAlign: 'center',
 
-    color: tokens.colorNeutralForeground2,
```

Comment:
It looks like the `subtle` and `strong` appearances were relying on this as the color being set here as they don't have overrides for `color`. I think they need to have this color added to their styles.

Reply:
Done!